### PR TITLE
Update RichHandler to include all flags of LogRender

### PR DESCRIPTION
**Pull request by nathanrpage97**
## Type of changes

- [ ] Bug fix
- [x] New feature
- [ ] Documentation / docstrings
- [ ] Tests
- [ ] Other

## Checklist

- [x] I've run the latest [black](https://github.com/ambv/black) with default args on new code.
- [x] I've updated CHANGELOG.md and CONTRIBUTORS.md where appropriate.
- [ ] I've added tests for new code.
- [x] I accept that @willmcgugan may be pedantic in the code review.

## Description

I saw a reference to this issue in #72. 

Update the Console and RichHandler class to include show_time and show_level.

### CODE REVIEW

File: rich/logging.py [hunk 1]
```diff
@@ -56,7 +58,9 @@ def __init__(
         super().__init__(level=level)
         self.console = console or get_console()
         self.highlighter = highlighter or self.HIGHLIGHTER_CLASS()
-        self._log_render = LogRender(show_level=True, show_path=show_path)
+        self._log_render = LogRender(
```
Comment (willmcgugan) [inline rich/logging.py:61]: The `LogRender` class is also use in Console when you call `console.log` -- which doesn't have a `level` argument.

Comment (nathanrpage97) [reply rich/logging.py:61]: Ok, I've removed the log level from the creation of a console.

**Comment by willmcgugan** [comment]
Thanks

**Comment by ewels** [comment]
Nice, thanks both! I was about to come and request this feature 😅

**Pull request merged**